src/Symfony/Component/HttpClient/RetryableHttpClient.php

```diff
@@ -60,7 +60,7 @@ public function request(string $method, string $url, array $options = []): Respo
         return new AsyncResponse($this->client, $method, $url, $options, function (ChunkInterface $chunk, AsyncContext $context) use ($method, $url, $options, &$retryCount, &$content, &$firstChunk) {
             $exception = null;
             try {
-                if ($chunk->isTimeout() || null !== $chunk->getInformationalStatus() || $context->getInfo('canceled')) {
+                if ($context->getInfo('canceled') || $chunk->isTimeout() || null !== $chunk->getInformationalStatus()) {
```

Comment:
This change is just an performance boost?

Reply:
It's mostly to be 200% sure that canceled requests won't take the "catch" code-path, in case `isTimeout()` accidentally throw. It should not for canceled requests, but better be defensive.

Reply:
Thx for explaination.